lib/gpt/core/util.py

```diff
@@ -87,3 +88,14 @@ def all_have_attribute(value, a):
     if type(value) == list and len(value) > 0:
         return all([all_have_attribute(v, a) for v in value])
     return hasattr(value, a)
+
+
+# fdimensions
+def fdimensions_from_openqcd(fdim):
```

Comment:
fdimensions_to/from_openqcd should not go into util.py .  We should never need them apart from a single test file.  They need to go into the test.

Reply:
Resolved in https://github.com/lehner/gpt/pull/79/commits/0242ad51a0d92431f27be78634de6541dbd727b7.